app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -64,6 +64,19 @@ def connect(options = {})
     self.class.raw_connect(server, port, username, password, service)
   end
 
+  def metrics_connect(options = {})
+    equire 'ovirt_metrics'
```

Comment:
The fact that the tests pass with this typo, means that we need better test coverage.